app/src/main/java/org/sopt/tabling/presentation/queue/QueueReservationFragment.kt

```diff
@@ -27,13 +31,67 @@ class QueueReservationFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        val reservationAdapter = ReservationAdapter(requireContext())
-        binding.rvReservationItems.adapter = reservationAdapter
-        reservationAdapter.setReservationList(viewModel.mockReservationList)
+        setReservationList()
     }
 
     override fun onDestroyView() {
         super.onDestroyView()
         _binding = null
     }
+
+    private fun setReservationList() {
+        ServicePool.reservationListService.getReservationList()
+            .enqueue(object : retrofit2.Callback<ResponseReservationDto> {
+                override fun onResponse(
+                    call: Call<ResponseReservationDto>,
+                    response: Response<ResponseReservationDto>,
+                ) {
+                    if (response.isSuccessful) {
+                        val data: ResponseReservationDto = response.body()!!
+                        setReservationAdapter(data.reservationList)
+                        if (response.code() == 500) {
+                            context!!.showToast("서버 에러 발생")
+                        }
+                    }
+                }
+
+                override fun onFailure(call: Call<ResponseReservationDto>, t: Throwable) {
+                    context!!.showToast("네트워크 에러 발생")
```

Comment:
```suggestion
                    Log.e("network", t.message.toString())
```

이런 식으로 토스트 메시지를 이용하는 게 아니라 내려오는 오류 값을 로그를 통해 찍어보면 어떤 것 때문에 실패했는지 알 수 있어서 좋아요